inc/model.class.php

```diff
@@ -797,16 +803,14 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
             switch ($item->getType()) {
                 case __CLASS__:
                     $tabs[1] = __('Model');
-                    if (!$this->isNewID($item->fields['id'])) {
+                    if (!$this->isNewID($item->fields['id'])) { /** @phpstan-ignore-line */
```

Comment:
Maybe something like this (_ditto for similar cases_)? :
```suggestion
                    if (!$this->isNewID($item->fields['id'] ?? -1)) {
```

Reply:
when I remove the /** @phpstan-ignore-line */ phpstan gives me an error that CommonGLPI::$fields does not exist and that ?? is not nullable

![image](https://github.com/pluginsGLPI/datainjection/assets/107540223/315ab3f9-1865-4c80-872e-cf5cb45b8799)